code/modules/organs/organ.dm

```diff
@@ -281,12 +281,15 @@ var/list/organ_cache = list()
 	switch (severity)
 		if (1.0)
 			take_damage(0,20)
+			rejuvenate()
 			return
 		if (2.0)
 			take_damage(0,7)
+			rejuvenate()
 			return
 		if(3.0)
 			take_damage(0,3)
```

Comment:
I saw that and wasn't sure what to think. I should probably report weird things I see in the code more often.